contents/docs/cdp/batch-exports/index.md

```diff
@@ -32,12 +33,78 @@ A batch export is executed in batch runs depending on the configured frequency.
 
 As an example, creating a batch export of events with daily frequency today will schedule the first batch run to start right as tomorrow begins. Thus, the data exported are events that PostHog received from 00:00:00 until 23:59:59 of today.
 
-Batch runs are configured on the [exports page](https://app.posthog.com/exports) in your PostHog instance.
-
 > **Note:** When deciding which data falls within the bounds of a specific batch run, we look at the time when the data landed in PostHog's ClickHouse database. This means that network and processing delays can make an event that was sent within the bounds of a batch run fall in a future run.
 
-## How does this differ from the old export system?
+### Tracking progress
+
+On each batch export view we will be presented with a list of the latest runs executed:
+
+![batch export runs](../../../images/docs/batch-exports/batch-exports-runs.png)
+
+Each run has an indicator of the state (which can be either "Starting", "Running", "Failed", or "Completed"), the boundaries for the data exported (data interval start and data interval end), when the run actually started, and the option of retrying a specific run.
```

Comment:
```suggestion
Each run has:
1. A state indicator which can be either "Starting", "Running", "Failed", or "Completed."
2. The exported data start and end intervals.
3. When the run actually started.
4. The option of retrying a specific run.
```

Reply:
Nice, we may add more stuff later as we expose more elements in the UI, a list works best 👍